<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionAbout"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionGames"/>
    <addaction name="actionNotes"/>
    <addaction name="actionDifficulties"/>
    <addaction name="actionTags"/>
    <addaction name="actionReviews"/>
    <addaction name="actionStorage"/>
   </widget>
   <widget class="QMenu" name="menuStats">
    <property name="title">
     <string>Stats</string>
    </property>
    <addaction name="actionPredefined"/>
    <addaction name="actionSaved"/>
    <addaction name="actionNew_from_scratch"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuStats"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionGames">
   <property name="text">
    <string>Games</string>
   </property>
  </action>
  <action name="actionNotes">
   <property name="text">
    <string>Notes</string>
   </property>
  </action>
  <action name="actionDifficulties">
   <property name="text">
    <string>Dictionaries</string>
   </property>
  </action>
  <action name="actionTags">
   <property name="text">
    <string>Tags</string>
   </property>
  </action>
  <action name="actionReviews">
   <property name="text">
    <string>Reviews</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionStorage">
   <property name="text">
    <string>Storage</string>
   </property>
  </action>
  <action name="actionPredefined">
   <property name="text">
    <string>Predefined</string>
   </property>
  </action>
  <action name="actionSaved">
   <property name="text">
    <string>Saved</string>
   </property>
  </action>
  <action name="actionNew_from_scratch">
   <property name="text">
    <string>New from scratch</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
